data_acquisition/py/gee_functions.py

```diff
@@ -323,6 +302,121 @@ def remove_geo(image):
   return image.setGeometry(None)
 
 
+def apply_fill_mask(image):
```

Comment:
question: since this function gets applied before band names are harmonized, are we potentially missing B6 (Swir1) for ls89?

Reply:
This should absolutely be the case! Thank you. Will update.